pkgs/by-name/ap/apkeditor/jcommand/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  fetchFromGitHub,
+  gradle,
+  lib,
+  REAndroidLibrary,
+}:
+
+let
+  self = REAndroidLibrary {
+    pname = "jcommand";
+    version = "2024-09-20";
```

Comment:
The convention for unreleased unstable versions is `0-unstable-YYYY-MM-DD`

Reply:
Ah, I saw some examples popping up when I searched `version = "202"`, so I thought that was the convention.